azurerm/internal/services/securitycenter/security_center_automation_resource.go

```diff
@@ -128,9 +128,11 @@ func resourceSecurityCenterAutomation() *schema.Resource {
 							Type:     schema.TypeString,
 							Required: true,
 							ValidateFunc: validation.StringInSlice([]string{
-								"Alerts",
-								"Assessments",
-								"SubAssessments",
+								string(security.Alerts),
+								string(security.Assessments),
+								string(security.SecureScoreControls),
+								string(security.SecureScores),
+								string(security.SubAssessments),
```

Comment:
can we update the documentation to include these new values? this should also likely be case-sensitive (by changing the `true` below)

Reply:
sorry for missing. Have done it!